articles/cross-origin-authentication/index.md

```diff
@@ -7,27 +7,44 @@ description: An explanation of cross-origin authentication in Auth0 and its comp
 
 For most situations, Auth0 recommends that authentication transactions be handled at the [Hosted Login Page](/hosted-pages/login). Doing so offers the easiest and most secure way to authenticate users. It is, however, understood that some scenarios necessitate that the Lock widget or a custom login form be directly embedded in an application. Cross-origin authentication provides a way to do this securely.
 
+::: panel Limitations of Third-Party Cookies
```

Comment:
I would place this right after this paragraph https://github.com/auth0/docs/pull/5093/files#diff-cfa1cdc9d8180ddd9066f090a188ae3eR20 because the third-party cookies are introduced there. And if you do this you can remove some text from here, like the first sentence. Try to keep these panels short and to the point. For example, this is a bit too expansive: `The users must not only have a browser that supports third-party cookies, but in some cases, the user must also not have disable them (see the [browser testing matrix](#browser-testing-matrix)). If you wish to use cross-origin authentication, you must must ask your users to leave third party cookies enabled or to switch browsers.`. Just tell them what they need to be aware of